src/components/Display/Display.spec.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Display from './Display';
+
+describe('Calculator', () => {
+    let wrapper;
+    beforeEach(() => wrapper = shallow(<Display displayValue={'0'}/>))
```

Comment:
```suggestion
    let props;
    beforeEach(() => {
      props = {
        displayValue:'0'
      };
      wrapper = shallow(<Display {..props}/>));
    }
```
While you only have one prop, a convention we use often is creating a separate `prop` variable to store all props and then spread them onto the component. This will make updating individual properties easier and props overall more re-usable in the future.

In your current case, I think this is fine since you only have one instance of setting a prop and generating the shallowly rendered component. Just something to be aware of.